portal/src/graphql/portal/PasswordsScreen.tsx

```diff
@@ -47,8 +70,14 @@ const PasswordsScreen: React.FC = function PasswordsScreen() {
   }
 
   return (
-    <main className={cn(styles.root, { [styles.loading]: updatingAppConfig })}>
-      {updateAppConfigError && <ShowError error={updateAppConfigError} />}
+    <main
+      className={cn(styles.root, {
+        [styles.loading]: updatingAppAndEmailSmsTemplateConfig,
+      })}
+    >
+      {updateAppAndEmailSmsTemplateConfigError && (
+        <ShowError error={updateAppAndEmailSmsTemplateConfigError} />
```

Comment:
If there is `reason` in API error response, we can handle those error and show it in dialog? (using `ErrorDialog`)